lib/internal/test_runner/reporter/v8-serializer.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { DefaultSerializer } = require('v8');
+const { Buffer } = require('buffer');
+const { serializeError } = require('internal/error_serdes');
+
+
+module.exports = async function* v8Reporter(source) {
+  const serializer = new DefaultSerializer();
+
+  for await (const item of source) {
+    const originalError = item.data.details?.error;
+    if (originalError) {
+      // Error is overriden with a serialized version, so that it can be
+      // deserialized in the parent process.
+      // Error is restored after serialization.
+      item.data.details.error = serializeError(originalError);
```

Comment:
this isn’t guaranteed to succeed just because it was truthy - if it’s nonwritable or has a setter. What can `source` be?

Reply:
it is defined here https://github.com/nodejs/node/blob/24615bd409d148c9ef2d03d6706d0807d5df4557/lib/internal/test_runner/test.js#L717

Reply:
Looks good then, please resolve :-)